yb-voyager/cmd/importData.go

```diff
@@ -760,19 +738,23 @@ func getTablesToImport() ([]string, []string, []string, error) {
 		donePattern := fmt.Sprintf("%s/%s.%s.D", metaInfoDataDir, t, SPLIT_INFO_PATTERN)
 		interruptedPattern := fmt.Sprintf("%s/%s.%s.P", metaInfoDataDir, t, SPLIT_INFO_PATTERN)
 		createdPattern := fmt.Sprintf("%s/%s.%s.C", metaInfoDataDir, t, SPLIT_INFO_PATTERN)
+		lastSplitPattern := fmt.Sprintf("%s/%s.%s.[CPD]", metaInfoDataDir, t, LAST_SPLIT_PATTERN)
 
 		doneMatches, _ := filepath.Glob(donePattern)
 		interruptedMatches, _ := filepath.Glob(interruptedPattern)
 		createdMatches, _ := filepath.Glob(createdPattern)
+		lastSplitMatches, _ := filepath.Glob(lastSplitPattern)
 
```

Comment:
Exit if `len(lastSplitMatches) > 1`. This will let us know if there is any bug in the implementation.

Reply:
done

Reply:
done